front/src/components/changePasswordContainer/ChangePasswordContainer.tsx

```diff
@@ -1,5 +1,30 @@
 import React from 'react';
-import { Box, Button, FormControl, FormLabel, Heading, Input, useColorMode } from '@chakra-ui/core';
+import {
+  Box,
+  Button,
+  FormControl,
+  FormLabel,
+  Heading,
+  Input,
+  useColorMode,
+  useToast,
+} from '@chakra-ui/core';
+import axios from 'axios';
+import { API } from 'utils/constants';
+import { connect, MapStateToProps } from 'react-redux';
+import { AppState, AppThunkDispatch, AppThunkPlainAction } from 'store';
+import { selectUserEmail } from 'store/user/selectors';
+import { setIsLoggedOutAction } from 'store/user/actions';
+
+interface StateProps {
+  email: string;
+}
+
+interface DispatchProps {
+  setIsLoggedOut: AppThunkPlainAction;
```

Comment:
I don't see setIsLoggedOut being used - is it ok to remove?

Reply:
I originally intended to use it for protected routing, but it was already implemented, forgot to remove it. Yes, it can be removed.